notebooks/search/09-semantic-text.ipynb

```diff
@@ -0,0 +1,510 @@
+    "\n",
+    "To get started, we'll need to connect to our Elastic deployment using the Python client (version 8.15.0 or above).\n",
+    "Because we're using an Elastic Cloud deployment, we'll use the **Cloud ID** to identify our deployment.\n",
+    "\n",
+    "First we need to `pip` install the following packages:\n",
+    "\n",
+    "- `elasticsearch`"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "f9c8bd62c8241f90"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "!pip install elasticsearch"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "13fdf7656ced2da3"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Next, we need to import the modules we need. \n",
+    "\n",
+    "🔐 NOTE: getpass enables us to securely prompt the user for credentials without echoing them to the terminal, or storing it in memory."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "9d54b112361d2f3d"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from elasticsearch import Elasticsearch, exceptions\n",
+    "from urllib.request import urlopen\n",
+    "from getpass import getpass\n",
+    "import json"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "9a60627704e77ff6"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now we can instantiate the Python Elasticsearch client.\n",
+    "\n",
+    "First we prompt the user for their password and Cloud ID.\n",
+    "Then we create a `client` object that instantiates an instance of the `Elasticsearch` class."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "eb9498124146d8bb"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# https://www.elastic.co/search-labs/tutorials/install-elasticsearch/elastic-cloud#finding-your-cloud-id\n",
+    "ELASTIC_CLOUD_ID = getpass(\"Elastic Cloud ID: \")\n",
+    "\n",
+    "# https://www.elastic.co/search-labs/tutorials/install-elasticsearch/elastic-cloud#creating-an-api-key\n",
+    "ELASTIC_API_KEY = getpass(\"Elastic Api Key: \")\n",
+    "\n",
+    "# Create the client instance\n",
+    "client = Elasticsearch(\n",
+    "    # For local development\n",
+    "    # hosts=[\"http://localhost:9200\"]\n",
+    "    cloud_id=ELASTIC_CLOUD_ID,\n",
+    "    api_key=ELASTIC_API_KEY,\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "6e14437dcce0f235"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Enable Telemetry\n",
+    "\n",
+    "Knowing that you are using this notebook helps us decide where to invest our efforts to improve our products. We would like to ask you that you run the following code to let us gather anonymous usage statistics. See [telemetry.py](https://github.com/elastic/elasticsearch-labs/blob/main/telemetry/telemetry.py) for details. Thank you!"
```

Comment:
```suggestion
    "Knowing that you are using this notebook helps us decide where to invest our efforts to improve our products, we would like to ask you that you run the following code to let us gather anonymous usage statistics. See [telemetry.py](https://github.com/elastic/elasticsearch-labs/blob/main/telemetry/telemetry.py) for details. Thank you!"
```

Reply:
This is boilerplate copy used on all the notebooks, so I think we should keep it uniform